.ci/run_nccl_test.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/bash -leE
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+echo "DEBUG: SCRIPT_DIR = ${SCRIPT_DIR}"
+
+. ${SCRIPT_DIR}/settings.sh
+
+GLOBAL_TEST_STATUS=0
+
+if [ -z "${NCCL_DIR}" ]
+then
+    echo "ERROR: NCCL_DIR is not defined"
+    echo "FAIL"
+    exit 1
+fi
+
+if [ -z "${NCCL_TESTS_DIR}" ]
+then
+    echo "ERROR: NCCL_TESTS_DIR is not defined"
+    echo "FAIL"
+    exit 1
+fi
+
+#
+# Set affinity to 2 cores according to Jenkins executor number
+#
+# if [ -n "$EXECUTOR_NUMBER" ]
+# then
+    # AFFINITY="taskset -c $(( 2 * EXECUTOR_NUMBER ))","$(( 2 * EXECUTOR_NUMBER + 1))"
+# else
+    # AFFINITY=""
+# fi
+# echo "DEBUG: AFFINITY = $AFFINITY"
+
+HOSTFILE=${CI_DIR}/cfg/$HOSTNAME/hostfile
+NP=2
+IB_DEV="mlx5_0:1"
+MPIRUN_OPTIONS_COMMON="\
+-x LD_LIBRARY_PATH \
+-x NCCL_DEBUG=INFO \
+-x HCOLL_MAIN_IB=${IB_DEV} \
+-x NCCL_DEBUG_SUBSYS=INIT \
+-x NCCL_IB_HCA=${IB_DEV} \
+-x NCCL_NET_GDR_LEVEL=5 \
+-x NCCL_SOCKET_IFNAME=eno1 \
+-x UCX_NET_DEVICES=${IB_DEV} \
+-x HCOLL_ENABLE_SHARP=0 \
+-x HCOLL_ENABLE_MCAST_ALL=0 \
+-mca pml ucx \
+-mca btl self,tcp \
```

Comment:
why this needed? if pml ucx set - btl has no effect, can be removed.

Reply:
Fixed (copy-pasted from the original test scenarios provided by the plugin engineers).

Reply:
It seems tests hang without `-mca btl self,tcp`. I've temporarily returned it back - need to double check.

Reply:
Seems it was an irregular hanging issue not related to pml/btl (potentially cluster specific) - I'll check it additionally.